objects/ohitbox/Step_2.gml

```diff
@@ -158,11 +159,18 @@ if (collisionCheck)
 				{
 					collision_list[| i].owner.sprite_index = collision_list[| i].owner.CharacterSprites.hurt_Sprite;
 				} 
-				// Set the correct Sprite
-				collision_list[| i].owner.prevState = eState.HURT; // Set the victim's previous state to HURT
-				collision_list[| i].owner.state = eState.HITSTOP; // Set the victim's state to hitstop
-				collision_list[| i].owner.isShortHopping = false; // Make sure the victim is not using their shorthop fall speed.
 				
+				// Set the correct Sprite
+				if (!collision_list[| i].owner.isDestructibleObject) // Check if the hurbox is attatched to a destructible object
+				{
+					collision_list[| i].owner.prevState = eState.HURT; // Set the victim's previous state to HURT
+					collision_list[| i].owner.state = eState.HITSTOP; // Set the victim's state to hitstop
+					collision_list[| i].owner.isShortHopping = false; // Make sure the victim is not using their shorthop fall speed.
+				}
+				else
+				{
+					collision_list[| i].owner.state = eState.HITSTOP;

```

Comment:
Remove this else clause, and put this line on it's own after the previous if. End result should be:

```
// Set the correct Sprite
if (!collision_list[| i].owner.isDestructibleObject) // Check if the hurbox is attatched to a destructible object
{
	collision_list[| i].owner.prevState = eState.HURT; // Set the victim's previous state to HURT
	collision_list[| i].owner.isShortHopping = false; // Make sure the victim is not using their shorthop fall speed.
}
				
collision_list[| i].owner.state = eState.HITSTOP; // Set the victim's state to hitstop
```

Reply:
Destructible objects don't have a hurt state, only a hitstop state.